Music.py

```diff
@@ -338,7 +416,126 @@ def rebuild_sequences(rom, sequences):
             rom.write_byte(base, j.instrument_set)
 
 
-def rebuild_pointers_table(rom, sequences):
+    # Patch new instrument sets (banks) and add new instrument sounds
+    added_banks = [] # Store copies of all of the banks we've added
+    added_instruments = [] #Store copies of all of the instruments we've added
+    new_bank_index = 0x26
+    instr_data = bytearray(0) # Store all of the new instrument data that will be added to the end of audiotable
+
+    audiobank_start, audiobank_end, audiobank_size = rom.get_dmadata_record_by_key(0xD390)
+    audiotable_start, audiotable_end, audiotable_size = rom.get_dmadata_record_by_key(0x79470)
+
+    instr_offset_in_file = audiotable_size
+    for i in range(0x6E):
+        bank_table_base = rom.sym('AUDIOBANK_TABLE_EXTENDED')
```

Comment:
`rom.sym` is not safe in cosmetics code since the addresses can change between versions. You'd want to put the address to this in the cosmetics context and retrieve it from the `symbols` dictionary passed to `patch_music` in `Cosmetics.py`. Additionally, it should handle versions from before the new ASM hacks, similar to the minimum version check for custom music to be enabled in `randomize_music`.

Reply:
So do we need to support both versions of custom music? As is, this code removes the support for the old format.

Reply:
Ideally it should support patching older versions with sequences that don't have custom instruments/banks. I haven't looked too thoroughly at code to know how feasible that is. Worst case scenario is that it should just disable custom music if the symbol isn't in the cosmetics context so that patching older versions doesn't straight trash the ROM.

Reply:
So supporting sequences without custom instruments/banks should be doable but they will have to be in the zip format.

Reply:
That's absolutely okay. .meta/.seq outside of ZIPs is fine to die, though we'll probably want to put the documentation/script to easily convert them somewhere that support can easily point to.

Reply:
Is there a format to the value used for COSMETIC_FORMAT_VERSION in config.asm/Cosmetics.py or do I just increment it by 1?

Reply:
We've just incremented it lately. There's really no format to it. I think the initial version was chosen to just be a random enough set of bytes that is unlikely to occur by accident, but I could be wrong. I think I randomly changed a few digits in it once.

Reply:
That might do it :)